libs/books/feature/src/lib/book-search/books-grid/books-grid.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { Store } from '@ngrx/store';
+import {
+  ReadingListBook
+} from '@tmo/books/data-access';
+
+@Component({
+  selector: 'tmo-books-grid',
+  templateUrl: './books-grid.component.html',
+  styleUrls: ['./books-grid.component.scss']
```

Comment:
Could you change the `ChangeDetectionStrategy` here?

Reply:
changing to `ChangeDetectionStrategy.OnPush`